src/converters/replugged.ts

```diff
@@ -24,7 +24,7 @@ export function convertFormat(ast: Statement[]) {
         targetClassName = exportedClass.id?.name;
         break;
     }
-    if (targetClassName != undefined) {
+    if (!(targetClassName === undefined)) {
```

Comment:
1. These are triple equals
2. WHY LIKE THIS
3. Just do `if (targetClassName !== undefined) {`

Reply:
ffs